### PR TITLE
solve issue #138

### PR DESCRIPTION
**Pull request by gamaanderson**

### CODE REVIEW

**Comment by jjhelmus** [comment]
@gamaanderson Thanks for the quick turn around on this.  I'm glad to have this issues addressed.  

Two very minor fixes before this is merged.  There is a violation of the [pep8](https://www.python.org/dev/peps/pep-0008/) style guidelines on line 262 which we are following in Py-ART, line 588 also should have an extra space if you want to fix that at the same time.  Line 456 contains tailing whitespace which is a no-no, I have my editor set to strip trailing whitespace on .py files to prevent this.

I can fix these two issues after merging if you would prefer, just let me know.

File: pyart/map/grid_mapper.py [hunk 1]
```diff
@@ -256,6 +259,7 @@ def find_neighbors_and_dists(self, q, r):
 
 
 def map_to_grid(radars, grid_shape, grid_limits, grid_origin=None,
+                grid_origin_alt = None,
```
Comment (jjhelmus) [inline pyart/map/grid_mapper.py]: There should not be spaces around the = sign, grid_origin_alt=None

File: pyart/map/grid_mapper.py [hunk 2]
```diff
@@ -442,7 +452,8 @@ def map_to_grid(radars, grid_shape, grid_limits, grid_origin=None,
         radar_lat = float(radar.latitude['data'])
         radar_lon = float(radar.longitude['data'])
         x_disp, y_disp = corner_to_point(grid_origin, (radar_lat, radar_lon))
-        offsets.append((0, y_disp, x_disp))  # XXX should include z
+        z_disp = float(radar.altitude['data']) - grid_origin_alt
+        offsets.append((z_disp, y_disp, x_disp))
```
Comment (jjhelmus) [inline pyart/map/grid_mapper.py]: Tailing whitespace, should be removed.

**Comment by gamaanderson** [comment]
I will solve this. I'm not used with playing that much attention to style and I also follow a some what different style. So I'm sorry, I will try to avoid in the future, but this will probably happen again.

**Comment by jjhelmus** [comment]
Do not worry about style too much, I fix up a lot of style issues in pull requests and an happy to do it.  There is a command line tool that I used to check for style prior to checking in files that you might find useful, [pep8](https://pypi.python.org/pypi/pep8).

**Comment by gamaanderson** [comment]
what is the problem in line 588? I can't see it

**Comment by jjhelmus** [comment]
There should be a space between # and the first letter.  It has been in there for quite some time, pep8 only started picked up the error a few months back.

**Comment by gamaanderson** [comment]
in line 262 I've put grid_origin_alt in a line of its own, breaking the overall block, should I also correct that?

**Comment by jjhelmus** [comment]
Would be good, it is not a technical violation it just looks out of place.

**Comment by jjhelmus** [comment]
Merged, FYI there was a long line that I fixed up pre-merge ca611ddd9e

**Pull request merged**